tests/helpers/__init__.py

```diff
@@ -0,0 +1,2 @@
+from tests.helpers.boring_model import BoringDataModule, BoringModel, RandomDataset  # noqa: F401
+from tests.helpers.datasets import TrialMNIST  # noqa: F401
```

Comment:
```suggestion
from tests.helpers.datasets import TrialMNIST  # noqa: F401

```

I think this is failing pep8